crates/wasi-http/tests/all/main.rs

```diff
@@ -527,6 +527,29 @@ async fn do_wasi_http_echo(uri: &str, url_header: Option<&str>) -> Result<()> {
     Ok(())
 }
 
+#[test_log::test(tokio::test)]
+async fn wasi_http_without_port() -> Result<()> {
+    do_http_request_without_port().await
+}
+
+async fn do_http_request_without_port() -> Result<()> {
+    let request = hyper::Request::builder()
+        .method(http::Method::GET)
+        .uri("http://example.com/test-path");
```

Comment:
Could you add an additional request for a path that will return a 200?

Reply:
Sure, for some reason this is returning `404` in the CI, any idea why?